packages/i18n/turbo.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": ["//"],
+  "tasks": {
+    "dev": {
+      "cache": false,
+      "persistent": true
+    },
+    "build": {
+      "dependsOn": [],
+      "outputs": ["dist/**"]
+    },
+    "lint:js": {
+      "inputs": ["src/**/*.{js,mjs,ts,tsx}"]
+    },
+    "lint:md": {
+      "inputs": ["src/**/*.md?(x)"],
```

Comment:
what is that we didn't have any src directory ?

Reply:
It is a mistake :D